[2025-10-06T02:15:07.935+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-06T02:15:07.944+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab2.task_using_linked_owner manual__2025-10-06T02:15:05.760088+00:00 [queued]>
[2025-10-06T02:15:07.946+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab2.task_using_linked_owner manual__2025-10-06T02:15:05.760088+00:00 [queued]>
[2025-10-06T02:15:07.946+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-06T02:15:07.950+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task_using_linked_owner> on 2025-10-06 02:15:05.760088+00:00
[2025-10-06T02:15:07.954+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=177) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-06T02:15:07.954+0000] {standard_task_runner.py:63} INFO - Started process 189 to run task
[2025-10-06T02:15:07.955+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab2', 'task_using_linked_owner', 'manual__2025-10-06T02:15:05.760088+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpns00al0a']
[2025-10-06T02:15:07.956+0000] {standard_task_runner.py:91} INFO - Job 121: Subtask task_using_linked_owner
[2025-10-06T02:15:07.970+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab2.task_using_linked_owner manual__2025-10-06T02:15:05.760088+00:00 [running]> on host 1f6a89bf0e15
[2025-10-06T02:15:07.999+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Devadarshini' AIRFLOW_CTX_DAG_ID='Airflow_Lab2' AIRFLOW_CTX_TASK_ID='task_using_linked_owner' AIRFLOW_CTX_EXECUTION_DATE='2025-10-06T02:15:05.760088+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-06T02:15:05.760088+00:00'
[2025-10-06T02:15:08.000+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-06T02:15:08.005+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-06T02:15:08.005+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 1']
[2025-10-06T02:15:08.009+0000] {subprocess.py:86} INFO - Output:
[2025-10-06T02:15:08.011+0000] {subprocess.py:93} INFO - 1
[2025-10-06T02:15:08.011+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-06T02:15:08.011+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-06T02:15:08.019+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab2, task_id=task_using_linked_owner, run_id=manual__2025-10-06T02:15:05.760088+00:00, execution_date=20251006T021505, start_date=20251006T021507, end_date=20251006T021508
[2025-10-06T02:15:08.055+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-06T02:15:08.067+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-06T02:15:08.068+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
